Makefile.am

```diff
@@ -20,8 +20,13 @@ srpm:
 	$(MAKE) dist
 	rpmbuild $(RPM_DIRS) -bs $(PACKAGE_NAME).spec
 
+test:
+	cd test/ && $(MAKE) test coredump=$(coredump) executable=$(executable) package=$(package) release=$(release)
+
 # value for overriding from command line
 # e.g: make scratch-build DIST=f15
 DIST=rawhide
 scratch-build: srpm
 	koji build --scratch dist-$(DIST) `make srpm | grep Wrote | cut -d' ' -f2`
+
+.PHONY: all rpm srpm test scratch-build
```

Comment:
Why some many new targets? Don't you need only "test"? The .PHONY targets are useful when you want to force `make` to always execute the target - (the problem here is that the 'test' directory already exists and has no dependencies, so make will not the relevant code, therefore you must add 'test' to .PHONY targets - the reset of the targets can be removed).